kubernetes/hypertrace.sh

```diff
@@ -76,22 +112,90 @@ if [[ "$HT_ENABLE_DEBUG" == "true" ]]; then
+          python pre_post_install_task_generator.py post ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/install-manifests.yaml ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/post-install-tasks/post-install-manifests.yaml
+          echo "[INFO] post-install task manifests are generated at: " ${HYPERTRACE_HOME}"/platform-services/helm-deployment-templates/pre-install-tasks/pre-install-manifests.yaml"
+        else
+          mkdir ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/service-manifests/
+          tar -xvf ${HYPERTRACE_HOME}/platform-services/charts/$servicename* -C ${HYPERTRACE_HOME}/platform-services/charts/ 
+          if [ $HT_DATA_STORE == "postgres" ]; then
+            helm template hypertrace-platform-services ${HYPERTRACE_HOME}/platform-services/charts/$servicename -f ${HYPERTRACE_HOME}/platform-services/values.yaml -f ${HYPERTRACE_HOME}/platform-services/postgres/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/service-manifests/$servicename-manifests.yaml
+          else
+            helm template hypertrace-platform-services ${HYPERTRACE_HOME}/platform-services/charts/$servicename -f ${HYPERTRACE_HOME}/platform-services/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/platform-services/helm-deployment-templates/service-manifests/$servicename-manifests.yaml
+          fi
+          echo "[INFO]" $servicename " manifests are generated at: " ${HYPERTRACE_HOME}"/platform-services/helm-deployment-templates/service-manifests/"$servicename"-manifests.yaml"
+        fi
+        cleanup
+
+    elif [[ "$servicename" == "" ]]; then
+      echo "[INFO] creating helm deployment template for hypertrace data services."
+      helm dependency update ${HYPERTRACE_HOME}/data-services ${HELM_FLAGS}
+      if [ $HT_DATA_STORE == "postgres" ]; then
+        helm template hypertrace-data-services ${HYPERTRACE_HOME}/data-services -f ${HYPERTRACE_HOME}/data-services/values.yaml -f ${HYPERTRACE_HOME}/data-services/postgres/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/data-services/helm-deployment-templates/manifests.yaml
+      else
+        helm template hypertrace-data-services ${HYPERTRACE_HOME}/data-services -f ${HYPERTRACE_HOME}/data-services/values.yaml -f ${HYPERTRACE_HOME}/clusters/$HT_PROFILE/values.yaml --set htEnv=${HT_ENV} > ${HYPERTRACE_HOME}/data-services/helm-deployment-templates/manifests.yaml
+      fi
+      echo "[INFO] data services manifests are generated at: " ${HYPERTRACE_HOME}"/data-services/helm-deployment-templates/manifests.yaml"
+      echo "[INFO] creating helm deployment template for hypertrace platform services."
+      helm dependency update ${HYPERTRACE_HOME}/platform-services ${HELM_FLAGS}
+      if [ $HT_DATA_STORE == "postgres" ]; then
```

Comment:
You already have one function for doing similar thing `create_helm_install_manifests`,

Can we have two functions?
```
create_helm_install_manifests_for_data_services
create_helm_install_manifests_for_platform_services
```
This way, it would reduce copy/paste.